next.config.js

```diff
@@ -120,6 +120,50 @@ module.exports = withSourceMaps({
     // !! WARN !!
     ignoreBuildErrors: true,
   },
+  async redirects() {
+    return [
+      {
+        source: '/signup',
```

Comment:
Is there a special order here? Otherwise please sort these links alphabetically so we have better check the list if it's getting bigger in the future.